.github/workflows/cmsis_rv2.yml

```diff
@@ -0,0 +1,201 @@
+name: RTOS2 Validation
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches: [main]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  ARM_UBL_ACTIVATION_CODE: ${{ secrets.ARM_UBL_ACTIVATION_CODE }}
+
+jobs:
+  build-and-run:
+    strategy:
+      fail-fast: true
+      matrix:
+        compiler: [AC6, GCC, Clang]
+    
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - working-directory: /home/runner
+        env:
+          GH_TOKEN: ${{ github.token }}
+        run: |
+          if [ -d CMSIS_6 ]; then
+            cd CMSIS_6
+            git fetch origin main
+            git checkout -f origin/main
+          else
+            gh repo clone ARM-software/CMSIS_6
+          fi
+
+      - working-directory: /home/runner
+        env:
+          GH_TOKEN: ${{ github.token }}
+        run: |
+          if [ -d CMSIS-RTOS2_Validation ]; then
+            cd CMSIS-RTOS2_Validation
+            git fetch origin main
+          else
+            gh repo clone ARM-software/CMSIS-RTOS2_Validation
+            cd CMSIS-RTOS2_Validation
+          fi
+          git checkout -f 5c27a1f
+
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'    
+
+      - name: Install system packages
+        run: |
+          sudo add-apt-repository ppa:deadsnakes/ppa
+          sudo apt-get install libpython3.9 libtinfo5
+
+      - name: Python requirements
+        run: |
+          pip install -r /home/runner/CMSIS-RTOS2_Validation/Project/requirements.txt
+
+      - name: Cache packs
+        uses: actions/cache@v3
+        with:
+          key: packs-${{ github.run_id }}-${{ matrix.compiler }}-${{ matrix.rtos }}
+          restore-keys: |
+              packs-
+          path: /home/runner/.cache/arm/packs
+        
+      - name: Cache vcpkg
+        uses: actions/cache@v3
+        with:
+          key: vcpkg-${{ runner.os }}-${{ runner.arch }}-${{ github.run_id }}-${{ matrix.compiler }}-${{ matrix.rtos }}
+          restore-keys: |
+              vcpkg-${{ runner.os }}-${{ runner.arch }}-
+          path: /home/runner/.vcpkg
+
+      - name: Prepare vcpkg env
+        working-directory: /home/runner/CMSIS-RTOS2_Validation/Project
+        run: |  
+          . <(curl https://aka.ms/vcpkg-init.sh -L)
+          vcpkg x-update-registry --all
+          vcpkg activate
+          
+          echo "Fix library search path for VHT models"
+          echo "LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$(dirname $(which VHT_MPS2_Cortex-M3))" >> $GITHUB_ENV
+
+          echo "Fix execute permissions for VHT models"
+          chmod +x $(dirname $(which VHT_MPS2_Cortex-M3))/VHT_*
+
+          echo "Patch CMSIS-Toolbox"
```

Comment:
These patches are not necessary since cmsis-toolbox 2.10 was released

Reply:
Yes, this PR is slightly outdated and needs updates anyway. We can revive it once issues are solved.

Reply:
Removed